frontend/package.json

```diff
@@ -42,6 +42,7 @@
     "react-app-rewire-svg-react-loader": "codebandits/react-app-rewire-svg-react-loader",
     "react-app-rewired": "^1.5.2",
     "react-test-renderer": "^16.4.0",
+    "url-search-params": "^1.1.0",
```

Comment:
Apparently this package [has been deprecated](https://www.npmjs.com/package/url-search-params) and the author recommends to use [@ungap/url-search-params instead](https://www.npmjs.com/package/@ungap/url-search-params)